src/main/java/core/mate/academy/service/BulldozerProducerImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducerImpl<M> implements MachineProducer<M> {
+    @Override
+    public List<M> get() {
+        M bulldozer1 = (M) new Bulldozer();
+        M bulldozer2 = (M) new Bulldozer();
+        M bulldozer3 = (M) new Bulldozer();
+        return List.of(bulldozer1, bulldozer2, bulldozer3);
+    }
+}
```

Comment:
Perhaps it is worth using an array to add objects to the arraylist and add a constant in MachineProducer for flexible quantity correction.  Add it to all MachineProducer implementations

```suggestion
}
```    public List<Bulldozer> get() {
        List<Bulldozer> bulldozers = new ArrayList<>();
        for (int i = 0; i < MACHINE_QUANTITY; i++) {
            bulldozers.add(new Bulldozer());
        }
        return bulldozers;
    }
```suggestion
}
```